controllers/discovery/initDiscoveryUserData.js

```diff
@@ -69,11 +75,21 @@ const InitDiscoveryUserData = async (req, res) => {
     });
     let suggestedUsers = usersWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The execution of the `totalDataQuery` is done without error handling. If the query fails for some reason (e.g., database connection issues), the application will crash. It would be better to wrap this in a try-catch block to handle potential errors gracefully.



<!-- This is an auto-generated comment by OSS CodeRabbit -->